docs/shadow_config_spec.md

```diff
@@ -661,6 +662,33 @@ environment:
 environment: { ENV_A: "1", ENV_B: foo }
 ```
 
+#### `hosts.<hostname>.processes[*].expected_final_state`
```

Comment:
Also adding a comment here that tornettools and the benchmark runner's tor and tgen simulations will need to be updated.

Reply:
If I merge a benchmark change that sets `expected_final_state`, it'll break due to the config option being unknown until this PR is merged, won't it?

If I merge this PR first, the benchmarks will be broken until I merge changes to them.

I could be careful and avoid having a temporary broken state, but it's probably not worth the extra dev overhead vs just minimizing the time window that has the broken state

Reply:
Okay I might have misunderstood. Yeah I think it would be best to not worry about a rigorous solution, just it's probably best for the benchmark to be updated the same day as this PR is merged to avoid breaking the nightly benchmarks. You shouldn't have to update tornettools in order to fix the benchmark to work (just updating the benchmark yaml is enough), but updating tornettools sooner rather than later would probably be good so that it doesn't block us from updating tornettools in the benchmarks if we need to in future PRs.

Reply:
https://github.com/shadow/benchmark/pull/7 is q'd up to address this